### PR TITLE
Polish WebServiceTemplateBuilder

### PR DESCRIPTION
**Pull request by nosan**
This PR fixes `javadoc` and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/WebServiceTemplateBuilder.java [hunk 1]
```diff
@@ -520,27 +520,28 @@ private WebServiceMessageSenders(boolean onlyAdditional, Set<WebServiceMessageSe
 			this.messageSenders = messageSenders;
 		}
 
-		public boolean isOnlyAdditional() {
+		boolean isOnlyAdditional() {
```
Comment (philwebb) [inline spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/WebServiceTemplateBuilder.java]: I don't think we've yet defined a good code style for these kinds of methods. Obviously there's nothing wrong with dropping `public` but I don't think we're very consistent across the codebase. We should probably come up with a standard and start to enforce it with checkstyle.

Comment (philwebb) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/WebServiceTemplateBuilder.java]: We discussed this today and like the style that you've adopted. Going forward, we'll not have unnecessary public methods.

Comment (nosan) [reply spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/WebServiceTemplateBuilder.java]: Thanks

**Comment by wilkinsona** [comment]
Thanks very much, @nosan.

**Pull request closed without merging**